server/lib/parties/party-service.ts

```diff
@@ -242,25 +260,25 @@ export default class PartyService {
   }
 
   private subscribeToParty(clientSockets: ClientSocketsGroup, party: PartyRecord) {
+    this.userClients = this.userClients.set(clientSockets.userId, clientSockets.clientId)
+
     clientSockets.subscribe<PartyInitEvent>(
       getPartyPath(party.id),
       () => ({
         type: 'init',
         party: toPartyJson(party),
       }),
-      sockets => this.handleClientQuit(sockets),
+      sockets => this.removeClientFromParty(sockets),
     )
   }
 
-  private handleClientQuit(clientSockets: ClientSocketsGroup) {
+  private removeClientFromParty(clientSockets: ClientSocketsGroup) {
     const party = this.getClientParty(clientSockets)
     if (!party) {
       logger.error('error while handling client quitting, party not found')
```

Comment:
Missed this in previous reviews, but I'd create an Error when logging this and include it, so that we get a stack trace on this